Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionControllerCocoa.mm

```diff
@@ -275,6 +275,14 @@
         if (!context->hasPermission(frameURL))
             continue;
 
+        for (auto& styleSheet : context->dynamicallyInjectedUserStyleSheets()) {
+            auto page = WebProcessProxy::webPage(pageID);
```

Comment:
The return value of `WebProcessProxy:webPage()` might be a nullptr. You should either RELEASE_ASSERT that this will never happen, or do a nullptr check here (which is the typical pattern for this return type).

Reply:
Actually, `pageID` doesn't change in this loop, so it would be better to get its value outside the loop, and just use the resulting Controller inside.

E.g.,
```
if (auto* page = WebProcessProxy::webPage(pageID)) {
    WebUserContentControllerProxy& controller = page.get()->userContentController();
    for (auto& styleSheet : context->dynamicallyInjectedUserStyleSheets())
        controller.removeUserStyleSheet(styleSheet);
}
```